README.md

```diff
@@ -18,8 +18,8 @@ Action requires the following parameters:
    * `api-key-secret` - APS api key secret
    * `app_file` -  mobile application file (_.zip_, _.apk_ or _aab_)
 
-APS key id and APS key secret can be created and retrieved in [Verimatrix App Shield](https://appshield.verimatrixcloud.net/Developer)
-portal under **My Account -> Developer** menu.  
+APS key id and APS key secret can be created and retrieved in [Verimatrix XTD and Counterspy](https://portal.platform.verimatrixcloud.net/)
```

Comment:
Just leave out APS altogether... "Key ID and key secret". Also, I'd say "XTD _or_ Counterspy".

Reply:
Updated.